cmd/frontend/graphqlbackend/webhooks.go

```
@@ -33,8 +36,13 @@ func (r *webhookResolver) UUID() string {
 }
 
 func (r *webhookResolver) URL() string {
-	// TODO: Combine external URL and UUID
-	return "TODO"
+	externalURL, err := url.Parse(conf.Get().ExternalURL)
+	if err != nil {
+		// The external URL in the site config should never be invalid
+		return ""
```

I know that this should be impossible to get here (URL is parsed before being updated in code host config), but if impossible happens, should we at least log it?

For now this is totally a blind side, silently returning an empty string, which can be troublesome for our clients to debug if they somehow (?!?!) see it

---

Good point, maybe we should return an error in this case actually.

---

Just thought about that as well, going to change it to `(string, error)`

---

Weirdly difficult to get `url.Parse` to fail haha